macros/edr/materializations/tests/handle_failed_test_result_count.sql

```diff
@@ -0,0 +1,63 @@
+{% macro handle_failed_test_result_count(flattened_test) %}
+  {% set result_count_query = elementary.get_failed_test_result_count_query(flattened_test) %}
+  {% if result_count_query %}
+    {% set result_count = run_query(result_count_query).columns[0].values()[0] %}  
+  {% else %}
+    {% set result_count = none %}
+  {% endif %}
+  {% do elementary.get_cache('elementary_test_failed_count').update({model.unique_id: result_count}) %}
+{% endmacro %}
+
+{% macro get_failed_test_result_count_query(flattened_test) %}
+  {% set failed_count_config = elementary.get_common_test_failed_count_config(flattened_test) %}
+  {% set test_meta = flattened_test['meta'] %}
+  {% if 'elementary_failed_count' in test_meta %}
```

Comment:
I'm not sure this name would be clear for the user.
What do you think about `fail_count_expression`?

Reply:
changed to `failed_count_expression` as it is really more easy, understandable and versitile, as to your point about `fail_calc`, it seems to not exactly match our use case here